app/league/templates/dashboard/dashboard.html

```diff
@@ -19,7 +19,7 @@ <h2 class="sub-header">Players</h2>
           <tbody>
             {% for player in players -%}
             <tr>
-              <td>{{ player.aga_id }}</td>
+                <td><a href="players/{{player.id}}"> {{ player.aga_id }}</a></td>
```

Comment:
What do you think of making the whole row, or at least the name, part of the link?